### ISSUE TITLE
enaml-native-cli won't install using conda.

### ISSUE THREAD

**Issue by T0M-W** [codelv/enaml-native-cli#9]
When I try to run the install command it says that the package cannot be found. I installed conda correctly but it won't install. Am I doing something wrong?
![conda not installing](https://user-images.githubusercontent.com/38615817/39204018-d420ee80-47f6-11e8-9d8f-a60036f1080f.PNG)

**Comment by frmdstryr** [comment]
No, you're doing it right. It seems conda doesn't provide some of the packages on windows.

Instead you can install with `pip install https://github.com/codelv/enaml-native-cli/archive/master.zip`

The windows support is not the best at the moment (working on it).  So please let me know any issues you run into. Thanks!

**Comment by T0M-W** [comment]
It worked. Thanks for the quick reply. I love what you're doing. I have been searching how to make an app with python for the longest time. Keep it up!

Just a quick question, can I make this using Pycharm IDE? I saw that you used Pycharm in one of your video's. I just want to make sure if I can.

**Comment by frmdstryr** [comment]
Any editor will work. 

I use android-studio and install the python plugin so it has both android and python support in the same IDE.

**Comment by T0M-W** [comment]
Thanks. I'll try it out.
